__tests__/test-GitRemoteConnection.js

```diff
@@ -0,0 +1,251 @@
+/* eslint-env node, browser, jasmine */
+const {
+  GitRemoteConnection,
+  GitSideBand
+} = require('isomorphic-git/internal-apis')
+const bufferToStream = require('buffer-to-stream')
+const pify = require('pify')
+const concat = require('simple-concat')
+const stream = require('stream')
+
+/*
+A diagram might be helpful.
```

Comment:
`Server` is misaligned by one column to the left! 😢 *Important*

Reply:
Believe it or not, this is actually a mistake resulting from my editor font (Fira Code iScript) being slightly off of a true monospace. 😢